### ISSUE TITLE
Own nickname

### ISSUE THREAD

**Issue by riley-martine** [Alex-Rose/fb-messenger-cli#130]
Is it possible for the cli to display one's own nickname?

**Comment by marcel-valdez** [comment]
Yes, it is possible, we have all the information required to do it available, but it requires work, like anything else, but I do believe that with the new GraphQL data source it should be easier to "set up the data structures needed for this".
